### PR TITLE
add python support for oms_exportSSVTemplate

### PR DESCRIPTION
**Pull request by arun3688**
### Related Issues

https://github.com/OpenModelica/OMSimulator/issues/824

### Purpose

This PR implements python support to  `oms_exportSSVTemplate`  API

### CODE REVIEW

File: doc/UsersGuide/source/api/exportSSVTemplate.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#CAPTION#
+exportSSVTemplate
+--------------
```
Comment (lochel) [inline doc/UsersGuide/source/api/exportSSVTemplate.rst:3]: `---`

**Review by lochel — CHANGES_REQUESTED** [review]

File: doc/UsersGuide/source/api/exportSSVTemplate.rst (hunk 1 above)
Comment (arun3688) [reply doc/UsersGuide/source/api/exportSSVTemplate.rst]: fixed with https://github.com/OpenModelica/OMSimulator/pull/864/commits/08d59dce891dbd9bb98a4905c6f7c402aac5ddb0

**Comment by robha67** [comment]
I tested the python api command oms.exportSSVTemplate("model", "template.ssv") with the latest nightly windows minGW build. However, I get the following error: 'OMSimulator' object has no attribute 'exportSSVTemplate'.
Did I missunderstand you @arun3688, is the python api for this ticket not yet inplemented?

**Comment by arun3688** [comment]
@robha67 It is implemented, but the submodule is not merged with OpenModelica master i guess, So it is not available with the nightly builds

**Comment by robha67** [comment]
I am not using the OpenModelica Nightly build. I tried with the MinGW 64bit OMSimulator build (OMSimulator-mingw64-v2.0.0.post258-g3135d7a.zip).

**Comment by arun3688** [comment]
@robha67 I think you are using `mingw binaries` and trying to execute in `windows` i guess,  Please download the windows binaries
https://build.openmodelica.org/omsimulator/win-msvc64/OMSimulator-win64-v2.0.0.post258-g3135d7a.zip and test it , I tested myself and it works.

**Pull request merged**